tests/Magento_Page_Loading.php

```diff
@@ -0,0 +1,473 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+class MagentoPageLoading extends Sauce\Sausage\WebDriverTestCase
+{
+    protected $base_url = 'http://127.0.0.1/';
+
+    public static $browsers = [
+        [
+            'browserName' => 'chrome',
+            'desiredCapabilities' => [
+                'version' => '45.0',
+                'platform' => 'OS X 10.10',
+            ],
+        ],
+    ];
+
+    const ADMIN_USERNAME    = 'denis';
```

Comment:
please replace it with **user** which will be used only for **test** purposes
such functionality should be moved into separate config.php file